README.md

```diff
@@ -116,46 +116,56 @@ nginx:
 ## To Use with PCF Dev
 To setup `stratos-metrics` instance against [PCF Dev](), save the following to a file called `pcf.yaml`
 ```
-env:
-    CLUSTER_ADMIN_PASSWORD: admin
-    UAA_CF_IDENTITY_ZONE: uaa
-    DOMAIN: local.pcfdev.io
-    UAA_ADMIN_CLIENT_SECRET: admin-client-secret
-    UAA_HOST: uaa.local.pcfdev.io
-    UAA_PORT: 443
-    DOPPLER_PORT: 443
-firehoseExporter:
-    noIdentityZone: true
+consoleVersion: 2.0.0
+metrics:
+    enabled: true
```

Comment:
These should not be nested in `metrics`. All properties should be top-level. `consoleVersion` needs to be removed.